ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/bucketising/CoordinateConverterFactory.java

```diff
@@ -8,6 +10,9 @@ public static CoordinateConverter getCoordinateConverter(String projection) {
 		if ("lambert72".equalsIgnoreCase(projection)) {
```

Comment:
Maybe this could also be a constant. Or you could create an ENUM with the supported coordinate conversions : "noop", "lambert72".